### PR TITLE
Address all reviewers comments

### PR DESCRIPTION
**Pull request by MichalisPanayides**
## Comments addressed on this PR so far:
- Change vocabulary to be more application-focused ✔️
- Restructure the entire paper ✔️
- Add diagram of the game ✔️
- Define normal form game explicitly ✔️
- Address comment 5 by reviewer 3 ✔️
- Truncation effect ✔️
- Create response to reviewers ✔️

### CODE REVIEW

File: sections/conclusion/main.tex [hunk 1]
```diff
@@ -1,58 +1,82 @@
...
+\end{itemize}
+Although our research is motivated by the particular EMS-ED example, our 
+developed modelling framework and behavioural insights has application to 
+similar systems across a range of sectors and settings.
+The queueing model can be applied to any setting where individuals may be
+blocked on a separate queue.
+An example of such setting can be any type of delivery service where customers
+can purchase goods either online or walk in the store.
+More specifically consider a food delivery service where customers can order
+food online or walk in the store.
```
Comment (drvinceknight) [inline sections/conclusion/main.tex]:
```suggestion
```

(repetition)

File: sections/conclusion/main.tex [hunk 2]
```diff
@@ -1,58 +1,82 @@
...
-In fact, modelling behaviour in a healthcare setting is a much more complex 
+In fact, modelling behaviour in a healthcare setting is a much more complex
 task where some cooperation may often be observed.
-Another extension would be to explore the behaviour of the ED staff via an 
-agent-based model. 
-This in turn can be used to model ED staff as agents each with their own 
+Another extension would be to explore the behaviour of the ED staff via an
+agent-based model.
+This in turn can be used to model ED staff as agents each with their own
 behavioural traits.
```
Comment (drvinceknight) [inline sections/conclusion/main.tex]:
```suggestion
This in turn can be used to model emergent behaviour based on
assumptions of individual behavioural traits of ED staff.
```

File: sections/model_overview/main.tex [hunk 3]
```diff
@@ -1,197 +1,163 @@
...
 
 Here, \(\alpha\) represents the ``importance'' of each objective,
 where high \(\alpha\) indicates a higher weight on the proportion of lost 
 individuals and smaller \(\alpha\) a higher weight on the time blocked. 
+Note here that the choice of \(p_A\) and \(p_B\) rely solely on equation
+\ref{eq:obj_distributor}.
+An extension that can be applied here is the use of unobservable queues.
+The ambulance service does not necessarily know at which state hospitals A and
+B are at, and thus may need to make decisions based on other
+information~\cite{unobservablequeue}.
```
Comment (drvinceknight) [inline sections/model_overview/main.tex]: I think this is a bit confused: currently the model **is** an unobservable model: everything is based on rates and expectations.

File: sections/model_overview/numerical_results/main.tex [hunk 4]
```diff
@@ -293,81 +226,81 @@ \subsection{Numerical results}\label{sec:numerical_results}
 \(\tilde{B}\) of matrix \(B\) only on the column that is most frequently played
 (matrix \(A\): row \(5\), matrix \(B\): column \(6\), see Figure 
 \ref{fig:ard_penalty}).
-Note that the values of the payoff matrices only differ on the fourth digit
-which is the raw utility of the strategies played.
+Note that for the presentation of data, a scaling has been applied to obtain
+the values of the payoff matrices.
+The results are not affected by this scaling.
```
Comment (drvinceknight) [inline sections/model_overview/numerical_results/main.tex]:
```suggestion
Note that for the presentation of data, an affine transformation has been applied to obtain
the values of the payoff matrices.
The results are not affected by this scaling.
```

Comment (drvinceknight) [reply sections/model_overview/numerical_results/main.tex]: We should specifically write what the transformation is (in brackets somewhere, it's not super important).

**Review by drvinceknight — CHANGES_REQUESTED** [review]

File: sections/model_overview/main.tex (hunk 3 above)
Comment (MichalisPanayides) [reply sections/model_overview/main.tex]: Oh, I think I misinterpreted this issue #25. Will change this accordingly

**Pull request merged**